src/borg/archiver.py

```diff
@@ -940,7 +940,8 @@ def item_content_stream(item):
             """
             Return a file-like object that reads from the chunks of *item*.
             """
-            chunk_iterator = archive.pipeline.fetch_many([chunk_id for chunk_id, _, _ in item.chunks])
+            chunk_iterator = archive.pipeline.fetch_many([chunk_id for chunk_id, _, _ in item.chunks],
+                                                         is_preloaded=True)
```

Comment:
this is the fix for what was reported in #5568 - full archive extraction was used there.

Reply:
Ah I see.

Reply:
With this small change, RAM usage was much less (half of what it was before) and the file where the issue was before got through. Thanks!

Reply:
ok, great, so I'll merge this now.